src/Facets/MayanFacet.sol

```diff
@@ -264,6 +264,16 @@ contract MayanFacet is
                 // 0x2337e236 FastMCTP::createOrder(address,uint256,uint256,uint32,uint32,(bytes32,[*bytes32*],uint64,uint64,uint64,uint64,uint64,bytes32,uint16,bytes32,uint8,uint8,bytes32))
                 receiver := mload(add(protocolData, 0xe4))
             }
+            case 0xe27dce37 {
+                // solhint-disable-next-line max-line-length
```

Comment:
I think we removed the max-line-length rule altogether, didnt we? In that case we can remove the solhint disable comments